INSTALL.md

```diff
@@ -7,116 +7,135 @@
 Installing OpenCoarrays
 =======================
 
-[![Download as PDF][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf)
+[The following link is commented until it can be updated. To uncomment delete the backslashes, the ourtermost square brackets, and the trailing colon and hash mark]:#
+[\[\!\[Download as PDF]\][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf)]:#
```

Comment:
@rouson Please don't comment this out: This is a service that I setup that automatically generates the PDFs from the markdown pages when they are posted to GitHub, so it is never out of date! The best thing to do, assuming that 51a047f53110b0c28f944ec0b2c917c283b9b84c is still the HEAD commit on `edit-install-doc` would be the following:

```bash
git checkout edit-install-doc
git reset --hard ^51a047f53110b0c28f944ec0b2c917c283b9b84c # drop this commit by resetting to its parent (thats what the ^ carat indicates)
git push --force origin edit-install-doc
```

This will delete all commits after 51a047f53110b0c28f944ec0b2c917c283b9b84c's parent commit (so don't do this if you have committed more local changes) and reset HEAD to be 51a047f53110b0c28f944ec0b2c917c283b9b84c's parent commit, i.e., 28054a6c7e75e5de756a9f2f107853792d764883